muxpi/parse_image.sh

```diff
@@ -32,16 +32,16 @@ ARCHVARIANT="$1"
 DUT_IP="$2"
 NEPTUNE_IMAGE="$3"
 
-MUXPIPATH="$HOME/muxpi"
+MUXPIPATH="$(pwd)/muxpi"
 IMAGESPATH="$MUXPIPATH/images"
 
 rm -rf $IMAGESPATH
 
 VARIANT_MINIMAL="core-image-pelux-minimal-dev"
-URL_MINIMAL="https://pelux.io/jenkins/job/pelux-manifests_NIGHTLY_intel-minimal/lastSuccessfulBuild/artifact/artifacts_intel/core-image-pelux-minimal-dev-intel*/*zip*/artifacts_intel-minimal.zip"
+URL_MINIMAL="https://pelux.io/jenkins/job/pelux-manifests_NIGHTLY_$ARCHVARIANT-minimal/lastSuccessfulBuild/artifact/artifacts_$ARCHVARIANT/core-image-pelux-minimal-dev*/*zip*/artifacts_$ARCHVARIANT-minimal.zip"
 
 VARIANT_NEPTUNE="core-image-pelux-qtauto-neptune-dev"
-URL_NEPTUNE="https://pelux.io/jenkins/job/pelux-manifests_NIGHTLY_intel-qtauto/lastSuccessfulBuild/artifact/artifacts_intel-qtauto/core-image-pelux-qtauto-neptune-dev-intel-corei7*/*zip*/artifacts-qtauto.zip"
+URL_NEPTUNE="https://pelux.io/jenkins/job/pelux-manifests_NIGHTLY_$ARCHVARIANT-qtauto/lastSuccessfulBuild/artifact/artifacts_$ARCHVARIANT-qtauto/core-image-pelux-qtauto-neptune-dev*/*zip*/artifacts-qtauto.zip"
```

Comment:
consider to replace NEPTUNE with QTAUTO, but perhaps the tests depends on the fact that Neptune 3 UI is running?

Reply:
The URL_NEPTUNE stands only for the image name, its not really related to the tests. AFAIK, for now we run the same tests on qtauto image and minimal image. Probably this will change in the future tho. @sergeevaeka

Reply:
What I mean, are the tests dependent on running Neptune? What if another type of UI is used? Are the tests still valid?
Or are the tests unique for QtAuto?

Reply:
It is just name of image with UI, no dependencies on Neptune, just link to image that is with Neptune.

Reply:
Leave it as is, don't change anything